src/main/java/seedu/duke/data/storage/Decoder.java

```diff
@@ -30,17 +30,8 @@
  * @author Wang Qin
  */
 public class Decoder {
-    /**
-     * Parses the specified save file to return a HashMap of Module objects.
-     *
-     * @param dataFileName
-     *  The name of the file to read from
-     * @return
-     *  The HashMap modulesMap
-     * @throws FileNotFoundException
-     *  When the file does not exist
-     */
-    public static HashMap<String, Module> loadModules(String dataFileName) throws FileNotFoundException {
+
+    public static HashMap<String, Module> loadModules(String dataFileName) {
```

Comment:
Curious why javadoc deleted? 1 more occurence below

Reply:
I deleted the exception throw. it is handled inside the func already